docs/how-to-help-with-video-challenges.md

```diff
@@ -136,6 +153,21 @@ question:
   solution: 3
 ```
 
+#### Example with markdown:
+````yml
+question:
+  text: |
+    What does this JavaScript code log to the console?
+    ```js
+    console.log(‘hello world’);
+    ```
+  answers:
+    - ‘hello’
+    - ‘world’
+    - ‘hello world’
```

Comment:
```suggestion
    - |
      mixing <a>html</a> with _markdown_ is allowed
```

Reply:
So in order to add HTML back in, I need to put another pipe(`|`) in there?

Reply:
Also, this suggestion removes the valid answer.

Reply:
I somehow completely missed that it was an actual question... I'm blind.

As for the pipe, no it's not necessary. It's just demonstrating that you can use it for answers as well as questions.  On reflection, it's probably trying to explain too much at once, so I'll change that.

Reply:
I think that's a little better.  Now it doesn't mess up your original plan (having an actual question), but we still show people a few things about the syntax.